hera_cal/reflections.py

```diff
@@ -442,85 +832,194 @@ def fit_reflection(data, dly_range, freqs, full_freqs=None, edgecut=0,
     # imag = np.array([interp1d(x, y[i].imag, kind='linear')(bin_shifts[i]) for i in range(Ntimes)])
     # ref_phs = np.angle(real + 1j*imag)
 
-    # construct reflection
-    if full_freqs is None:
-        full_freqs = freqs
-    eps = construct_reflection(full_freqs, ref_amps, ref_dlys, ref_phs)
+    return ref_phs
 
-    return (eps, ref_amps, ref_dlys, ref_phs, ref_dly_inds, ref_significance,
-            np.fft.fftshift(dfft, axes=1), np.fft.fftshift(dlys))
 
+def _svd_waterfall(data):
```

Comment:
This function isn't necessary. If you want to clarify the shapes of the PCs and svals, just add a comment to the one place where you use it.